tests/integration/test_sqs.py

```diff
@@ -136,6 +137,20 @@ def test_create_queue_and_get_attributes(self, sqs_client, sqs_queue):
         assert int(float(attrs["CreatedTimestamp"])) == pytest.approx(int(time.time()), 30)
         assert int(attrs["VisibilityTimeout"]) == 30, "visibility timeout is not the default value"
 
+    @pytest.mark.only_localstack
+    def test_queue_name_too_long(self, sqs_create_queue, snapshot):
+        with pytest.raises(InvalidParameterValue) as e:
+            sqs_create_queue(QueueName=(81 * "a"))
+
+        snapshot.match("queue_name_invalid_character", e.value.response)
```

Comment:
this seems like something we could test with snapshotting. here are a couple of examples. if you don't have an AWS account you want to test it with, then i'm happy to help with tests. https://github.com/localstack/localstack/blob/27b4e328cd2fbc3c7d3a11ee5d88c771da641d08/tests/integration/test_sqs.py#L1408-L1422